package.json

```diff
@@ -31,7 +31,7 @@
     "globby": "^9.1.0",
     "husky": "^1.3.1",
     "jest": "24.9.0",
-    "lerna": "3.18.4",
+    "lerna": "2.9.1",
```

Comment:
Had to downgrade this package as in lerna 3+ the ability to use `--scope` used for publishing has been removed so makes publishing our custom fork not possible

Reply:
We should bias towards keeping lerna up-to-date.  [See docs](https://github.com/lerna/lerna/tree/master/commands/publish#publishconfigaccess) as long as all other packages remain `"private": true` then lerna shouldn't publish em.